src/constants.ts

```diff
@@ -0,0 +1,10 @@
+const BACKEND_URL = 'https://lessoner.herokuapp.com';
+const EMAIL = {
+  min: 3,
+  max: 256,
+};
+const PASSWORD = {
+  min: 6,
```

Comment:
I think, it will be better to rename constants to EMAIL_LENGTH and PASSWORD_LENGTH

Reply:
I rename keys constants EMAIL={minLength:6, maxLength=256} also PASSWORD